component/Item/index.js

```diff
@@ -0,0 +1,58 @@
+import Link from "next/link";
+import {
+  StyledDetailSection,
+  StyledHeading,
+  StyledList,
+  StyledListContainer,
+  StyledRemainingDays,
+} from "../GroceriesList/GroceriesList.styled";
+import { groceries } from "@/resources/groceries.js";
+import { calculateDaysRemaining } from "@/resources/calculateDaysRemaining";
+import Navigation from "../Navigation";
+import { styled } from "styled-components";
+
+export default function Item({ id }) {
+  const grocerySelectedItem = groceries.find((item) => item.id === id);
+  console.log(id);
+  return (
+    <>
+      <StyledButton>
+        <StyledButtonLink href="/">⬅️ Back</StyledButtonLink>
+      </StyledButton>
```

Comment:
From a semantic point of view I would not mix up a `<button>` with `<a>`. Buttons are for interactions on the current page, links are there to navigate or switch pages.